Resources/doc/sync/descriptors/descriptors.rst

```diff
@@ -0,0 +1,122 @@
+Descriptors
+===========
+
+Extraction descriptors are special classes that tell extractor which fields/tables to watch for changes.
```

Comment:
There can be multiple, as long as they implement `ExtractionDescriptorInterface`.